src/anshuKJun20/SwapNumber.java

```diff
@@ -0,0 +1,22 @@
+package anshuKJun20;
+
+public class SwapNumber {
+
+	private void swapMethod(int number1, int number2) {
+		System.out.println("Input = ("+number1+" "+number2+")");
+		 number1 = number1*number2;
+		 number2 = number1/number2;
+	     number1 = number1/number2;
+		display(number1, number2);
+	}
+
+	void display(int number1, int number2) {
+		System.out.println("Output = ("+number1+" "+number2+")");
+	}
+
+	public static void main(String[] args) {
+		SwapNumber swapno = new SwapNumber();
```

Comment:
Follow camel case while writing variable name.The way you have written method name.
Instead of writing 'swapno' write 'swapNo' or give any other proper variable name

Reply:
changes Done .Next time i will take care.Thankyou for reviewing this.